src/pages/patientView/genomicOverview/tracksHelper.ts

```diff
@@ -56,11 +58,40 @@ function getChmEndsPerc(chms: Array<any>, total: any) {
 /**
  * storing chromesome length info
  */
-export function getChmInfo() {
-    let sel: any = {};
-    sel.hg19 = [0,249250621,243199373,198022430,191154276,180915260,171115067,159138663,146364022,141213431,135534747,135006516,133851895,115169878,107349540,102531392,90354753,81195210,78077248,59128983,63025520,48129895,51304566,155270560,59373566];
-    sel.total = 3095677412;
-    sel.perc = getChmEndsPerc(sel.hg19,sel.total);
+export const genomeBuilds: Map<string, string> = new Map([
+    [ "hg19", "GRCh37" ],
+    [ "37", "GRCh37" ],
+    [ "hg38", "GRCh38" ],
+    [ "38", "GRCh38" ],
+    [ "mm10", "GRCm38" ]
+]);
+
+export type ChromosomeSizes = {
+    genomeBuild: string,
+    chromosomeSize: number[]
+};
+
+const chromSizes:ChromosomeSizes[] = chromosomeSizes;
```

Comment:
(1) whats the point of having chromSizes when all it does is assign a reference to chromosomeSizes?